webapp/packages/core-app/src/shared/SessionExpire/SessionExpireService.ts

```diff
@@ -35,6 +35,11 @@ export class SessionExpireService {
   }
 
   private async sessionExpiredInterceptor(request: Promise<any>): Promise<any> {
+    if(this.isNotifiedAboutExpiredSession) {
+      const e = new Error('Session expired');
+      e.name = 'SessionError';
+      throw e;
+    }
```

Comment:
it's not how it works, in this case, interceptor executes for already run promise, so request to the backend already sent, but we want to block any requests after the expired session

so we probably need to add request block possibility to `CustomGraphQLClient`, we should block requests in `overrideRequest`